packages/protocol/contracts/bridge/Bridge.sol

```diff
@@ -666,7 +666,10 @@ contract Bridge is EssentialContract, IBridge {
 
             // Also need to add the cost of transferring a nonzero msg.value. This cost is currently
             // 9000, but provides a 2300 gas stipend to the called contract.
-            result_ = gasleft() * 63 < _minGas * 64 + (memoryGasCost + 9300) * 63;
+
+            // Also need to add the cost of the external CALL, which is 5700.
```

Comment:
Where does the 5700 gas come from? Just the call itself is 100 gas, and then there are some additional costs which are already taken into account I think, though we are pessimistic here because we don't check if there is a msg.value. A transfer to an empty account will cost an additional 25k gas, but because the call requires implementing an interface I think this case cannot be triggered.

I don't know how accurate this formula here needs to be.

Reply:
I think being more accurate is always better. I updated the code and comment and listed all extra cost that we need to take into consideration. Please take another look.